trophic_levels-biomass_indicators.rmd

```diff
@@ -123,12 +123,12 @@ names(totalherbivorebiomass)<-allyears_vect
 totalherbivorebiomass
 ggplot()+geom_sf(data=norcounty,fill="white",lwd=0.1)+
   geom_spatraster(data=totalherbivorebiomass)+facet_wrap(~lyr)+
-  scale_fill_continuous(trans='log10',low="lightskyblue",high="darkblue", na.value=NA,expression('Herbivore \nbiomass \nkg km'^-2 ))+theme_bw()
+  scale_fill_continuous(trans='log10',low="lightskyblue",high="darkblue", na.value=NA,expression(atop('Herbivore \nbiomass', 'kg km'^-2)))+theme_bw()
 
 
 ```
 
-And finally total carnivore biomass
+And finally total carnivore biomass:[?]
```

Comment:
Here the years are not visible in the headings of the 12 panels. Would be great to see them.